tests/plugins/easyimage/manual/scrollbarchromebug.md

```diff
@@ -0,0 +1,14 @@
+@bender-tags: 4.9.0, bug, 932
+@bender-ui: collapsed
+@bender-ckeditor-plugins: sourcearea, wysiwygarea, toolbar, easyimage, link
+
+----
+Note: Problem exists only in **Chrome** browsers. All other browsers should work fine and flickering scrollbar shouldn't be present.
+
+1. Move cursor over image in both editors.
+
+**Expected:** Nothing unusal happen in 1st editor. Unfortunatelly bug is not fixed in divarea editor, so flickering should be observable in bottom editor, what is expected behaviour here.
+
+**Unexpected:**
+  * Empty scrollbar shows up.
+  * Scrollbar flicker when cursor is moved over the images.
```

Comment:
I find this test description a little messy. First there is a note about Chrome (making it Chrome only would solve this one - see previous comment). Then **expected** saying that it shouldn't flicker in first editor but should in second. And then **unexpected** saying that it shouldn't flicker. It's a little bit confusing. 

Maybe something like:

> 1.  Move cursor over image in both editors.
>
> **Expected**: Nothing unusual happens, scrollbar is not flickering.
> 
> **Unexpected**: 
> * Empty scrollbar shows up.
> * Scrollbar flicker when cursor is moved over the images.
>
> **Note**: In **Chrome** browsers in second editor, scrollbars flickers as the issue is not fixed for `divarea` > editor (you may add link to upstream issue here).

If the test will be Chrome only you may adjust (shorten) description event more.